opennms-webapp/src/main/webapp/WEB-INF/jsp/event/detail.jsp

```diff
@@ -204,6 +225,28 @@
       </table>
     </div>
 
+    <c:if test="${showParms}">
+    <div class="card severity">
+      <div class="card-header">
+        <span>Parameters</span>
+      </div>
+      <% if (event.getParms() != null && event.getParms().size() > 0) { %>
+      <table class="table table-sm severity">
+        <% for (Map.Entry<String, String> parm : event.getParms().entrySet()) { %>
```

Comment:
Maybe sort the entries by key, so they appear always in the same order.

Reply:
So, actually, something that sometimes matters with event parameters is their order, like when you're trying to figure out what value to use in an event configuration for `/events/event/mask/varbind/vbnumber`. I'd argue it would make more sense to change the map implementation used when the `OnmsEvent` objects are converted to `Event`s from `HashMap` to `LinkedHashMap` to preserve insertion order, given that `OnmsEvent`'s parameters are a list. Objections?

Reply:
I could see that...